### ISSUE TITLE
Yarn audit is showing 3 high vulnerabilities

### ISSUE THREAD

**Issue by aidanw** [GeekyAnts/NativeBase#2796]
using `"native-base": "^2.12.2",` which is what was pulled when I did a fresh `yarn add native-base`

The output from `yarn audit` is this.
```
yarn audit v1.13.0
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Prototype Pollution                                          │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Package       │ lodash                                                       │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Patched in    │ >=4.17.12                                                    │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Dependency of │ native-base                                                  │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Path          │ native-base > lodash                                         │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ More info     │ https://nodesecurity.io/advisories/1065                      │     
└───────────────┴──────────────────────────────────────────────────────────────┘     
┌───────────────┬──────────────────────────────────────────────────────────────┐     
│ high          │ Prototype Pollution                                          │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Package       │ lodash                                                       │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Patched in    │ >=4.17.12                                                    │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Dependency of │ native-base                                                  │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Path          │ native-base > native-base-shoutem-theme > lodash             │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ More info     │ https://nodesecurity.io/advisories/1065                      │     
└───────────────┴──────────────────────────────────────────────────────────────┘     
┌───────────────┬──────────────────────────────────────────────────────────────┐     
│ high          │ Prototype Pollution                                          │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Package       │ lodash                                                       │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Patched in    │ >=4.17.12                                                    │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Dependency of │ native-base                                                  │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ Path          │ native-base > react-native-easy-grid > lodash                │     
├───────────────┼──────────────────────────────────────────────────────────────┤     
│ More info     │ https://nodesecurity.io/advisories/1065                      │     
└───────────────┴──────────────────────────────────────────────────────────────┘     
3 vulnerabilities found - Packages audited: 948389
Severity: 3 High
Done in 2.58s.
```

**Comment by ofekby** [comment]
Same in npm

**Comment by sankhadeeproy007** [comment]
Should be fixed in 2.13.5
